test/external-e2e/run.sh

```diff
@@ -16,6 +16,8 @@
 
 # Ensure that we have the desired version of the ginkgo test runner.
 
+set -xe
```

Comment:
I think only this line is needed.

Reply:
I think along with this, we should trap the function `print_logs` because without `TRAP` the script will stop abruptly without printing the logs.

Reply:
I have updated the PR, to remove the `print_logs` and `trap`. I think now we can verify if this is working as expected. thanks